fuse-products/src/main/java/software/tnb/product/integration/builder/AbstractIntegrationBuilder.java

```diff
@@ -565,4 +568,13 @@ public boolean isJBang() {
     public List<Resource> getBytemanRules() {
         return bytemanRules;
     }
+
+    public SELF addAdditionalPorts(Integer ... ports) {
+        additionalPorts.addAll(Arrays.stream(ports).toList());
+        return self();
+    }
+
+    public List<Integer> getAdditionalPorts() {
+        return additionalPorts;
+    }
```

Comment:
the additional ports handling also needs to be also implemented in `OpenshiftQuarkusApp` or these methods should be moved to `SpringBootIntegrationBuilder` as it will work only for springboot

Reply:
@avano I added the ports in the manual service generation, since it seems that it works out of the box, at least with jolokia, WDYT?